product_docs/docs/edbcloud/beta/getting_started/01a_managing_quotas.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Managing service limits and constraints in Azure
+
+---
+
+Your Azure subscription must meet certain quota (service limit) requirements to work with EDB Cloud. By default, Azure sets a very low limit on the number of VMs, vCPUs (cores), and public IP addresses permitted for a particular region in your subscription.
+
+To enable EDB Cloud to perform its operations successfully, ensure that the following requirements are met.  
+
+ * Is there enough quota for vCPU and IP addresses in your region?
+ * Are there enough virtual machine SKUs available in your region?
```

Comment:
This bullet doesn't make sense to me - it's not a matter of there being "enough" SKUs - a SKU is available at a certain point in time in a certain region, or it's not.

Reply:
Rephrased to:

"Is there sufficient limit on the number of vCPUs for the Standard Dsv2 and Standard Esv3 VM size families?"

Does this look OK?